pkg/backup/item_backupper.go

```diff
@@ -112,6 +114,33 @@ type defaultItemBackupper struct {
 
 var podsGroupResource = schema.GroupResource{Group: "", Resource: "pods"}
 var namespacesGroupResource = schema.GroupResource{Group: "", Resource: "namespaces"}
+var jobsGroupResource = schema.GroupResource{Group: "batch", Resource: "jobs"}
+
+// shouldBackup checks completed status on a given object and determines if the object should be backed up
+func shouldBackup(obj runtime.Unstructured, groupResource schema.GroupResource) (bool, string, error) {
```

Comment:
`shouldBackup` is a little ambiguous since there are lots of reasons why a resource should/shouldn't be backed up. Maybe `isCompletedResource`?